MaxMind.Db/Metadata.cs

```diff
@@ -39,24 +42,28 @@ public class Metadata
         /// <summary>
         ///     The MaxMind DB database type.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
         [JsonProperty("database_type")]
         public string DatabaseType { get; private set; }
 
         /// <summary>
         ///     A map from locale codes to the database description in that language.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
         [JsonProperty("description")]
         public Hashtable Description { get; private set; }
 
         /// <summary>
         ///     The IP version that the database supports. This will be 4 or 6.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
         [JsonProperty("ip_version")]
         public int IPVersion { get; private set; }
 
         /// <summary>
         ///     A list of locale codes for languages that the database supports.
         /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1811:AvoidUncalledPrivateCode")]
         [JsonProperty("languages")]
         public List<string> Languages { get; private set; }
 
```

Comment:
How is the setter private, uncalled according to analysis, yet required and in need of suppression?